.github/workflows/sanitized_ci.yml

```
@@ -161,7 +163,7 @@ jobs:
           .\make.bat -msvc
           .\v.exe self
       - name: Ensure code is well formatted
-        run: v test-cleancode
+        run: ./v test-cleancode
```

```suggestion
        run: .\v.exe test-cleancode
```

It is already fixed on master.